pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -49,18 +48,17 @@ import { GitHubLogo } from "../../assets/svgs/GitHub";
 import { DependenciesTemplate } from "../templateParts/dependenciesTemplates/ComponentDependenciesTemplate";
 import { useResultDisplayLayoutContext } from "../../context/resultDisplayLayout";
 import { ComponentCardsAccordionTemplate } from "../templateParts/componentCardsAccordion/ComponentCardsAccordionTemplate";
-import { DownloadTemplate } from "../templateParts/download/DownloadTemplate";
+import { DownloadProps, DownloadTemplate } from "../templateParts/download/DownloadTemplate";
```

Comment:
Usually we use the `...Props` keyword only for the props we're sending to a component. E.g. int he `DownloadTemplate` we have the `DownloadTemplateProps`, which are its props.

Now, we've added a `DownloadProps` interface which is actually not the main props for this component which might be confusing. I'd opt for making it a type, and naming it something along the lines of:

```
export type TDownloadableItem = { ... structure of type here }
```